.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+name: release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
```

Comment:
So much delicate config to duplicate across repos, at some point create our own github action? Is that even possible?

Reply:
Yes, but that will take time to develop and test.